src/components/providers/portfolio-provider.tsx

```diff
@@ -59,24 +67,63 @@ const PortfolioProvider: React.FC = ({ children }) => {
     }
   };
 
+  /**
+   * Loads history values
+   */
+  const loadHistoryValues = async () => {
+    try {
+      if (!statisticsLoaderParams) {
+        return;
+      }
+
+      const { effectivePortfolios, range } = statisticsLoaderParams;
+      const { startDate, endDate } = DateUtils.getDateFilters(range);
```

Comment:
Move things that do not throw errors outside try block

Reply:
Fixed